test/compliance/UPDATE.test.js

```diff
@@ -16,4 +21,18 @@ describe('UPDATE', () => {
       throw new Error('not supported')
     })
   })
+
+  describe('deep update', () => {
+    test('huge deep update does not cause sql exception', async () => {
+      // sqlite limit of placeholders is 32766 -> 9. of https://www.sqlite.org/limits.html
+      await INSERT.into('compositions.Travel').entries({
+        ID: 4711,
+        bookings: (new Array(33000).fill(0).map((_, i) => ({ ID: i})))
```

Comment:
Would it make sense to have a scaling number ? To identify the limits for all database implementations.